azure-pipelines.yml

```diff
@@ -7,21 +7,22 @@ pool:
   vmImage: 'Ubuntu 16.04'
 
 steps:
-- script: |
-    sudo npm install -g typescript
```

Comment:
no need to install anything globally - commands to run are npm install && npm test